staking_deposit/settings.py

```diff
@@ -1,37 +1,41 @@
 from typing import Dict, NamedTuple
-
+from eth_utils import decode_hex
 
 DEPOSIT_CLI_VERSION = '2.4.0'
 
 
 class BaseChainSetting(NamedTuple):
     NETWORK_NAME: str
     GENESIS_FORK_VERSION: bytes
+    GENESIS_VALIDATORS_ROOT: bytes
 
 
 MAINNET = 'mainnet'
-ROPSTEN = 'ropsten'
 GOERLI = 'goerli'
 PRATER = 'prater'
 SEPOLIA = 'sepolia'
 ZHEJIANG = 'zhejiang'
 
-
 # Mainnet setting
-MainnetSetting = BaseChainSetting(NETWORK_NAME=MAINNET, GENESIS_FORK_VERSION=bytes.fromhex('00000000'))
-# Ropsten setting
-RopstenSetting = BaseChainSetting(NETWORK_NAME=ROPSTEN, GENESIS_FORK_VERSION=bytes.fromhex('80000069'))
+MainnetSetting = BaseChainSetting(
+    NETWORK_NAME=MAINNET, GENESIS_FORK_VERSION=bytes.fromhex('00000000'),
+    GENESIS_VALIDATORS_ROOT=bytes.fromhex('4b363db94e286120d76eb905340fdd4e54bfe9f06bf33ff6cf5ad27f511bfe95'))
```

Comment:
Hello 👋 

I'm implementing credential change for a similar tool and came across this line when looking at your implementation. I'm really curious to know why you are changing the validators root and where you found this value. Won't this affect deposit messages as well? And lead to a different signing roots for deposits made with different versions of this tool?

I was under the impression that the genesis validators root was and will remain 0, but I'm having a hard time finding where these values are defined in the spec.

Reply:
Hi @alex-miller-0 

- `Deposit` operation doesn't use `genesis_validators_root` since the chain accepted deposits before genesis. 
- `BLSToExecutionChange` operation needs `genesis_validators_root` so that we can distinguish the chains with it.

From the specs, you can find it by how they call the [`compute_domain`](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#compute_domain) function.